relay-dynamic-config/src/metrics.rs

```diff
@@ -214,24 +314,25 @@ impl MetricExtractionConfig {
     /// The latest version for this config struct.
     ///
     /// This is the maximum version supported by this Relay instance.
-    pub const VERSION: u16 = 2;
+    pub const MAX_SUPPORTED_VERSION: u16 = 3;
```

Comment:
I renamed this constant because I always get confused by the `self.version <= Self::VERSION` check.